src/Command.ts

```diff
@@ -32,6 +32,17 @@ export interface CommandTypes {
   boolean?: string[];
 }
 
+export interface GraphResponseError {
```

Comment:
If this is specific to Graph, then it should be moved to the Graph base command class.